scripts/releaseInfo/index.js

```diff
@@ -116,8 +122,13 @@ const generateLaunchDates = service => {
       output.push(`__On Demand Radio__: ${serviceLaunch.onDemandRadio}`);
     }
 
+    if (serviceLaunch.idxPage && serviceLaunch.idxPage !== '') {
+      output.push(`__IDX__: ${serviceLaunch.idxPage}`);
+    }
+
     return output.join('<br/>');
   }
+  return null;
```

Comment:
Prior to this change, eslint did not like that we had a `{}` curly bracket function that had no return

![image](https://user-images.githubusercontent.com/30599794/86360563-785ae700-bc6a-11ea-9fe0-9de69be62ce8.png)